Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -602,7 +602,9 @@ static void logProcessPoolState(const WebProcessPool& pool)
     auto pool = extractWebProcessPool(observer);
     if (!pool)
         return;
-    pool->sendToAllProcesses(Messages::WebProcess::BacklightLevelDidChange(BKSDisplayBrightnessGetCurrent()));
+    float displayBrightness = BKSDisplayBrightnessGetCurrent();
+    pool->sendToAllProcesses(Messages::WebProcess::BacklightLevelDidChange(displayBrightness));
+    pool->m_displayBrightness = displayBrightness;
```

Comment:
Not clear to me what this change is about.

Reply:
This is caching the display brightness in the UI process, so we do not have to get this from the relevant service on each Web process launch. I will create a separate PR for this change.